rails_programming/introduction_to_rails/project_feet_wet.md

```diff
@@ -22,51 +22,54 @@ In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as
 </div>
 
 ### Student Solutions
-Send us your solution by editing this [file](https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/introduction_to_rails/project_feet_wet.md) so we can show others! Submit a link to the Github repo with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).
+Send us your solution by editing this [file](https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/introduction_to_rails/project_feet_wet.md) so we can show others! Submit a link to the Github repository with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).
 
-If you will be submitting a solution for the RoR tutorial chapter 2, make sure it's in a repo that you won't overwrite when we move onto chapter 3 (which starts a new app)
+Submit your solution for the Ruby on Rails tutorial chapter 2, make sure it is in a repository that you will not overwrite when we move onto chapter 3 (which starts a new app).
 
 <details markdown="block">
   <summary> Show Student Solutions </summary>
 
 * Add your solution below this line!
-* [David Auza's and Luis Novoa's Solution](https://github.com/luis-novoa/rails_toy_app) - [Live](https://shrouded-island-61851.herokuapp.com/)
-* [Andrew Huntington's Solution](https://github.com/AndrewHuntington/toy_app) - [Live](https://toy-app-54058.herokuapp.com/)
-* [Nasser Abachi's Solution](https://github.com/abachi/toy-app) - [Live](https://abachidev-toy-app.herokuapp.com/)
-* [Brendaneus' Solution](https://theodinprojects.live/courses/ruby-on-rails/projects/toy-app)
-* [BShowen's Solution](https://github.com/BShowen/rails_toy_app) - [Live](https://bradley-toy-app.herokuapp.com)
-* [Vedant's Solution](https://github.com/vedantshetty/Odin_Project_Code/tree/master/Ruby%20On%20Rails/toy_app) - [Live](https://fast-scrubland-64335.herokuapp.com/users)
-* [Ian's solution](https://github.com/IanMKesler/rails_toy_app) - [Live](https://whispering-journey-28034.herokuapp.com)
+<<<<<<< HEAD
```

Comment:
Remove the merge conflict marker

Reply:
There is another one that I'll also remove.